net-p2p/retroshare/retroshare-0.6.5.ebuild

```diff
@@ -0,0 +1,141 @@
+# Copyright 1999-2019 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+EGIT_REPO_URI="https://github.com/RetroShare/RetroShare.git"
+EGIT_COMMIT="v${PV}"
+EGIT_SUBMODULES=()
+
+inherit desktop eapi7-ver git-r3 gnome2-utils qmake-utils versionator
+
+DESCRIPTION="P2P private sharing application"
+HOMEPAGE="https://retroshare.cc"
+
+# pegmarkdown can also be used with MIT
+LICENSE="AGPL-3 GPL-2 GPL-3 Apache-2.0 LGPL-2.1"
+SLOT="0"
+KEYWORDS=""
```

Comment:
no keywords for release version?

Reply:
Because the source code is taken from git even if it is a release (in the tarball some parts of the build system are broken because it depends on git), CI complains if it has keywords

Reply:
but then what use is it for us?

Reply:
Taking the code from git tag v0.6.5 works well the user will just need to keyword it with `=net-p2p/retroshare-0.6.5 **`

Reply:
how is this an upgrade path from stable 0.6.4?

Reply:
Should be fixed now, thanks!